core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -47,39 +78,70 @@
     <background-banner></background-banner>
     <div ng-if="collection" class="oppia-collection-player-tiles-container">
       <h2 ng-if="!collectionPlaythrough.hasFinishedCollection()" class="oppia-page-heading">
-        <span ng-if="!collectionPlaythrough.hasStartedCollection()">Begin <[collection.getTitle()]>:</span>
+        <span ng-if="!collectionPlaythrough.hasStartedCollection()" style="font-family: Capriola, Roboto, Arial, sans-serif;">Begin <[collection.getTitle()]>:</span>
         <span ng-if="collectionPlaythrough.hasStartedCollection()">Continue <[collection.getTitle()]>:</span>
       </h2>
       <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-heading">
         <span>You have finished the collection! Feel free to replay any explorations below.</span>
       </h2>
 
       {% if is_logged_in %}
-        <div ng-if="collectionPlaythrough.getNextRecommendedCollectionNodeCount() > 0" class="oppia-collection-player-tile-section">
-          <div class="oppia-collection-player-small-text">
-            Next exploration<span ng-if="collectionPlaythrough.getNextRecommendedCollectionNodeCount() > 1">s</span> to play
-          </div>
-          <collection-node-list collection-id="collectionId" collection-nodes="getNextRecommendedCollectionNodes()"></collection-node-list>
-        </div>
+      <div ng-if="collection" class="exploration-list-table">
 
-        <div ng-if="collectionPlaythrough.getCompletedExplorationNodeCount() > 0" class="oppia-collection-player-tile-section">
-          <div class="oppia-collection-player-small-text">
-            Completed exploration<span ng-if="collectionPlaythrough.getCompletedExplorationNodeCount() > 1">s</span>
-          </div>
-          <collection-node-list collection-id="collectionId" collection-nodes="getCompletedExplorationNodes()"></collection-node-list>
-        </div>
+        <table class="oppia-dashboard-table">
+          <tr>
+            <th>
+              <p>Exploration</p>
+            </th>
+            <th class="hidden-md hidden-sm hidden-xs">
+              <p>Objective</p>
+            </th>
+          </tr>
+          <tr ng-repeat="collection in collection.getCollectionNodes()"
+          class="exploration-list-item">
+          <td>
+            <a ng-href="/explore/<[collection._explorationId]>"
+               ng-mouseover="updateExplorationPreview(collection._explorationId)"
```

Comment:
node.getExplorationId() may be better than accessing a private member. (If the method doesn't already exist, maybe add it?)